cumulus/tasks/pdr-status-check/tests/index.js

```diff
@@ -61,26 +64,35 @@ test('returns the correct results in the nominal case', (t) => {
     'arn:1': 'RUNNING',
     'arn:2': 'SUCCEEDED',
     'arn:3': 'FAILED',
-    'arn:4': 'ABORTED'
+    'arn:4': 'ABORTED',
+    'arn:7': null
+  };
+  const error = {
+    message: 'Execution Does Not Exist: arn',
+    code: 'ExecutionDoesNotExist'
   };
 
   const stubSfnClient = {
     describeExecution: ({ executionArn }) => ({
-      promise: () => Promise.resolve({
-        executionArn,
-        status: executionStatuses[executionArn]
-      })
+      promise: () => {
+        if (executionStatuses[executionArn] === null) return Promise.reject(error);
```

Comment:
This should be returning an actual Error instance, not a simple object.  Consider something like:

```
if (!executionStatuses[executionArn]) {
  const error = new Error(`Execution does not exist: ${executionArn}`);
  error.code = 'ExecutionDoesNotExist';
  Promise.reject(error);
}
```

Reply:
Thanks.  Done